internal/migrate/staging_validator.go

```diff
@@ -307,7 +307,7 @@ func (v *stagingValidator) loadSystemContracts() {
 		return
 	}
 
-	stagedSystemContracts := migrations.SystemContractChanges(chainId, migrations.SystemContractChangesOptions{
+	stagedSystemContracts := migrations.SystemContractChanges(chainId, migrations.SystemContractsMigrationOptions{
 		Burner: migrations.BurnerContractChangeUpdate, // needs to be update for now since BurnerChangeDeploy is a no-op in flow-go
 		EVM:    migrations.EVMContractChangeFull,
 	})
```

Comment:
It looks like this takes a `ChainID` field as well now.  Doesn't actually seem like it's used inside the function at all but we should probably add it just in case.